docs/layers.md

```diff
@@ -391,45 +391,39 @@ Places with `kind` values of `continent`, `country`, with others added starting
 
 * `name`
 * `id`: osm_id from OpenStreetMap or Natural Earth id
-* `kind`: the original value of the OSM `place` tag and Natural Earth `featurecla`
+* `kind`: normalizes values between OpenStreetMap and Natural Earth
 * `population`: population integer values from OpenStreetMap or Natural Earth (`pop_max`)
 * `scalerank`: scalerank value from Natural Earth, and invented for OpenStreetMap
 * `source`: `openstreetmap` or `naturalearthdata.com`
 
 #### Place properties (common optional):
 
 * `capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-0 capital`, `Admin-0 capital alt`, and `Admin-0 region capital`.
-* `state_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
+* `region_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
 * `labelrank`: labelrank value from Natural Earth
 * `min_zoom`: Currently neighbourhoods only, from Who's On First
 * `max_zoom`: Currently neighbourhoods only, from Who's On First
 * `is_landuse_aoi`: Currently neighbourhoods only, from Who's On First
+* `kind_detail`: the original value of the OSM `place` tag and Natural Earth `featurecla`, see below.
 
 #### Place kind values:
 
-* `Admin-0 capital alt`
-* `Admin-0 capital`
-* `Admin-0 region capital`
-* `Admin-1 capital`
-* `Admin-1 region capital`
-* `borough`
-* `city`
-* `continent`
 * `country`
+* `region`
+* `locality`
+* `neighbourhood`
+
+#### Place kind_detail values:
+
+* `city`
 * `farm`
 * `hamlet`
-* `Historic place`
 * `isolated_dwelling`
 * `locality`
-* `macrohood`
-* `Meteorological Station`
 * `neighbourhood`
-* `Populated place`
 * `province`
-* `quarter`
-* `Scientific station`
+* `scientific_station`
 * `state`
-* `suburb`
 * `town`
 * `village`
```

Comment:
@rmarianski , I updated the kind and kind_detail values with the ones that currently exist in the `places.yaml`. @nvkelso , can you check to make sure you agree with the changes?